src/reconstruction/comparison/modelBorgifier/src/buildRxnEquations.m

```diff
@@ -0,0 +1,103 @@
+% This file is published under Creative Commons BY-NC-SA.
```

Comment:
isn't this function essentially the output of "printRxnFormula", except with a minimally different style?